components/About/Page.js

```diff
@@ -197,12 +195,12 @@ const AboutPage = ({
           <Subheader2>Geschäftsbericht 2019–2020</Subheader2>
           <P {...styles.faqCta}>
             Zu Beginn{' '}
-            <a
+            <Editorial.A
               {...styles.link}
```

Comment:
`{...styles.link}` can and probably should be removed everywhere.

Reply:
jup, didn't catch that!

Reply:
done